glide.lock

```diff
@@ -667,4 +728,61 @@ imports:
   - tools/clientcmd/api
   - tools/metrics
   - transport
-testImports: []
+testImports:
```

Comment:
These imports are for the integration tests. This means we might be able to remove the integration vendor folder that way.
cc @timoreimann @ldez